doc/nrf/gs_toolchain.rst

```diff
@@ -0,0 +1,19 @@
+.. _gs_toolchain:
+
+Installing the |NCS| from a zip (experimental)
+##############################################
+
+.. warning::
+   Installing from a zip is not officially supported yet.
+   It should work fine for common use cases, but not all scenarios have been tested.
+
+To simplify the |NCS| installation on Windows operating systems, you can download a zip file that contains the full |NCS| toolchain.
```

Comment:
If this is available only for Windows, I would put a link to this page on Installing on Windows page OR mention Windows already in the title.